packages/ckeditor5-table/src/tablecolumnresize/utils.js

```diff
@@ -302,13 +290,15 @@ export function insertColumnResizerElement( viewWriter, viewCell ) {
 	);
 }
 
-// Calculates the total horizontal space taken by the cell. That includes:
-// * width;
-// * left and red padding;
-// * border width.
-//
-// @param {HTMLElement} domCell
-// @returns {Number} Width in pixels without `px` at the end
+/**
+ * Calculates the total horizontal space taken by the cell. That includes:
+ *  * width;
```

Comment:
```suggestion
 *  * width,
```

Reply:
Same as in the other rev comment, let's use commas for consistency with the rest of the docs.